### PR TITLE
fix for PR #40 inconsistency

### PR DESCRIPTION
**Pull request by devang-gaur**
I missed including the changes in the cico_run_tests.sh in PR #40 as part of issue #28

### CODE REVIEW

**Review by devang-gaur — COMMENTED** [review]
@anmolbabu have a look and please approve and merge. I'd be more careful next time.

File: cico_run_tests.sh [hunk 1]
```diff
@@ -1,12 +1,11 @@
 
 #!/bin/bash
 
-. cico_setup.sh
+. ./cico_setup.sh
```
Comment (anmolbabu) [inline cico_run_tests.sh:4]: Verified?

File: cico_run_tests.sh [hunk 2]
```diff
@@ -1,12 +1,11 @@
...
 
-TAG="test"
-TEST_IMAGE_NAME="$(make get-image-name):$TAG"
+TEST_IMAGE_NAME="$(make get-test-image-name)"
 
-./run_tests.sh
+. ./run_tests.sh
```
Comment (anmolbabu) [inline cico_run_tests.sh:8]: Why this change is required?

File: cico_run_tests.sh (hunk 1 above)
Comment (anmolbabu) [reply cico_run_tests.sh:4]: Does this statement import cico_setup.sh to satisfy push image line below?
Why do you want to run it from here using ./cico_setup.sh ?
I feel the previous state was itself correct

Comment (devang-gaur) [reply cico_run_tests.sh:4]: With  >" . cico_setup.sh" ,
I'm getting errors:

./cico_run_tests.sh: line 4: .: cico_setup.sh: file not found

File: cico_run_tests.sh (hunk 2 above)
Comment (dev-gaur) [reply cico_run_tests.sh]: Same reason why I changed to . ./cico_setup.sh .

Comment (devang-gaur) [reply cico_run_tests.sh:8]: To keep it on the line with   >". ./cico_setup.sh"  above on line 4.

**Review by anmolbabu — APPROVED** [review]

**Review by anmolbabu — APPROVED** [review]
